crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/controller/CrownCourtProceedingController.java

```diff
@@ -27,21 +26,21 @@
 @Slf4j
 @RestController
 @RequiredArgsConstructor
-@RequestMapping("api/internal/v1/crowncourtproceeding/actions")
+@RequestMapping("api/internal/v1/proceedings")
 @Tag(name = "Crown Court Proceeding", description = "Rest API for Crown Court Proceeding.")
 public class CrownCourtProceedingController {
 
-    private final CrownCourtProceedingService crownCourtProceedingService;
+    private final ProceedingService proceedingService;
 
-    private CrownCourtActionsRequestDTO preProcessRequest(ApiCheckCrownCourtActionsRequest crownCourtActionsRequest) {
-        return new CrownCourtActionsRequestDTOBuilder().buildRequestDTO(crownCourtActionsRequest);
+    private CrownCourtDTO preProcessRequest(ApiProcessRepOrderRequest request) {
+        return CrownCourtDTOBuilder.build(request);
     }
 
     @PostMapping(produces = MediaType.APPLICATION_JSON_VALUE)
-    @Operation(description = "Check Crown Court Actions data")
+    @Operation(description = "Process Crown Rep Order Data")
```

Comment:
Is this a typo? Is it meant to be "Process Rep Order Data" or "Process Crown Court Rep Order Data"?